src/v2/Apps/Auctions/Components/__tests__/AuctionArtworksRails.jest.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react"
+import { graphql } from "relay-runtime"
+import { setupTestWrapper } from "v2/DevTools/setupTestWrapper"
+import { AuctionArtworksRailFragmentContainer } from "../AuctionArtworksRail/AuctionArtworksRail"
+
+jest.unmock("react-relay")
+
+describe("AuctionArtworksRails", () => {
```

Comment:
#minor I think this got goofed up and should be singular like `AuctionArtworksRail` to match the component naming.

Reply:
Lol good catch.